modulemd/v2/include/modulemd-2.0/private/modulemd-util.h

```diff
@@ -49,3 +49,13 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC (modulemd_tracer, modulemd_trace_free);
   while (0)
 
 G_END_DECLS
+
+
+GHashTable *
+_modulemd_hash_table_deep_str_copy (GHashTable *orig);
```

Comment:
Let's drop the leading underscores. I don't know why I thought that was a good idea in v1, but I don't anymore :)